apps/cyberstorm-storybook/stories/pages/Communities.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import { StoryFn, Meta } from "@storybook/react";
+import CommunitiesLayout from "../../../cyberstorm-nextjs/app/communities/layout";
+import Page from "../../../cyberstorm-nextjs/app/communities/page";
+import React from "react";
+
+const meta = {
+  title: "Cyberstorm/Pages/Communities",
+  component: CommunitiesLayout,
+} as Meta<typeof CommunitiesLayout>;
+
+const Template: StoryFn<typeof CommunitiesLayout> = () => (
+  <div>
```

Comment:
Is the `div` needed? Methinks `<SomethingLayout>` should work as expected without it.